db/schema.rb

```diff
@@ -119,7 +119,10 @@
     t.text "status"
     t.text "vac_status"
     t.index ["course_id"], name: "IX_course_site_course_id"
+    t.index ["publish"], name: "index_course_site_on_publish"
```

Comment:
before we forget 

just maybe

https://www.postgresql.org/docs/9.6/indexes-multicolumn.html

```
  scope :findable, -> { status_running.published_on_ucas }


```
@timabell fyi

Reply:
Discussed offline. The individual indices will be fine here with this table size but this is a potential optimisation in the future.